lib/ReactGridLayout.jsx

```diff
@@ -780,17 +785,24 @@ export default class ReactGridLayout extends React.Component<Props, State> {
     this.dragEnterCounter++;
   };
 
-  onDrop = () => {
+  onDrop = (e: any) => {
```

Comment:
I tried to make the switch but during commit one of the tasks reports the following error:
"Cannot get e.dataTransfer because property dataTransfer is missing in Event".